test/oauth-server/service/httpApi.go

```diff
@@ -71,15 +73,25 @@ func NewRequestHandler(config *Config, idTokenKey *rsa.PrivateKey, accessTokenKe
 	if err != nil {
 		return nil, fmt.Errorf("cannot create jwk for idToken: %w", err)
 	}
+	authSession := cache.NewCache()
+	authRestriction := cache.NewCache()
+	refreshRestriction := cache.NewCache()
+	add := periodic.New(ctx.Done(), time.Second*5)
+	add(func(now time.Time) bool {
+		authSession.CheckExpirations(now)
+		authRestriction.CheckExpirations(now)
```

Comment:
Shouldn't also `refreshRestriction` be here?

Reply:
fixed